docs/root/operations/admin.rst

```diff
@@ -310,3 +310,38 @@ The fields are:
   Use the /runtime_modify endpoint with care. Changes are effectively immediately. It is
   **critical** that the admin interface is :ref:`properly secured
   <operations_admin_interface_security>`.
+  
+  .. _operations_admin_interface_hystrix_event_stream:
+
+.. http:get:: /hystrix_event_stream
+
+  Starts streaming statistics from envoy in 
+  `text/event-stream <https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events>`_ 
+  formatted stream for use by 
+  `Hystrix dashboard <https://github.com/Netflix-Skunkworks/hystrix-dashboard/wiki>`_. 
+  Hystrix dashobard is used for monitoring resiliency related metrics.
+
+  To start getting the stream from Envoy in Hystrix dashboard, this endpoint should be specified as 
+  its stream source.
+
+  This handler is enabled only when a Hystrix sink is enabled in the config file as documented
```

Comment:
I think you might want to note this at the top just under the /hystrix_event_stream:
1.  What you've noted here - that the handler is only enabled when the hystrix sink is enabled.
2.  That this GET won't work well (or at all) from a browser.  I think this will be the first admin handler that won't respond in a way that will work for a browser.

Reply:
Actually, when accessing the endpoint from a web browser, the stream is forwarded into a file that the browser asks to save, and is being continuously updated until Envoy stops the connection.

Reply:
I see.  Can you add a note about that browser behavior?